screens/rewards/RewardsScreen.js

```diff
@@ -73,6 +88,46 @@ export default class RewardsScreen extends React.Component {
       return null;
     }
 
+    if (this.state.isGuest) {
+      return (
+        <View style={{ flex: 1 }}>
+          <NavHeaderContainer vertical backgroundColor={Colors.primaryGreen}>
+            <NavButton onPress={() => this.props.navigation.goBack()}>
+              <FontAwesome5 name="arrow-down" solid size={24} color="white" />
+            </NavButton>
+            <BigTitle
+              style={{
+                marginLeft: 18,
+                color: Colors.lightest,
+                fontSize: 36,
+              }}>
+              Healthy Rewards
+            </BigTitle>
+          </NavHeaderContainer>
+          <ScrollView>
+            <Image
+              source={require('../../assets/images/guest.jpeg')}
```

Comment:
Oops I didn't catch this earlier but swap this out with the updated higher res png (Ace sent in Slack)